packages/api/package.json

```diff
@@ -17,7 +17,7 @@
     "prepublishOnly": "PRODUCTION=true npm run build"
   },
   "ava": {
-    "files": "tests/test-*.js",
```

Comment:
This will detect tests directly in the tests directory, as well as other subdirectories:

```
"ava": {
    "files": "tests/**/*.js",
    "serial": true
  }
```